src/main/java/com/cmsc495/ticketsystem/config/PasswordEncoderConfig.java

```diff
@@ -0,0 +1,20 @@
+/*Configuration file to encrypt password. 
```

Comment:
Looks good

Please add 

/* ITMS - A CMSC 495 Project
 * Group 2

^ See SubmitTicketController.java's header
 
 And if you don't mind please delete the following comment line from every header that has it, that part isn't necessary as anyone viewing the file in a VCS or code will be able to see the most recent commit that touched the file.
 
  * Last updated 14 SEP 24

Reply:
done and made it consistent across all the files